platform-core-module/src/main/java/com/softicar/platform/core/module/access/permission/ConvertedCoreModulePermission.java

```diff
@@ -0,0 +1,30 @@
+package com.softicar.platform.core.module.access.permission;
+
+import com.softicar.platform.common.core.i18n.IDisplayString;
+import com.softicar.platform.common.core.user.IBasicUser;
+import com.softicar.platform.core.module.module.instance.AGCoreModuleInstance;
+import com.softicar.platform.core.module.user.AGUser;
+import com.softicar.platform.emf.module.permission.IEmfModulePermission;
+import com.softicar.platform.emf.permission.IEmfPermission;
+
+class ConvertedCoreModulePermission<R> implements IEmfPermission<R> {
+
+	private final IEmfModulePermission<AGCoreModuleInstance> modulePermission;
+
+	public ConvertedCoreModulePermission(IEmfModulePermission<AGCoreModuleInstance> modulePermission) {
+
+		this.modulePermission = modulePermission;
+	}
+
+	@Override
+	public IDisplayString getTitle() {
+
+		return modulePermission.getTitle();
+	}
+
+	@Override
+	public boolean test(R tableRow, IBasicUser user) {
+
+		return AGUser.get(user).hasModulePermission(modulePermission, AGCoreModuleInstance.getInstance().pk());
```

Comment:
How certain are we about `AGUser.get` yielding non-null, here?

Reply:
100%...at least

Reply:
actually, this code was not touched, see `EmfConvertedModulePermission.java`